### ISSUE TITLE
Python 3: NameError: name 'unicode' is not defined

### ISSUE THREAD

**Issue by jtprobst** [signalfx/signalfx-python#31]
I saw that #29 was merged and 1.0.5 was released, so I tested it inside Docker (image python:latest) by running

```
pip3 install signalfx
```
Now I get the following error:
```
root@682c4a5e1844:/# python
Python 3.5.2 (default, Sep 24 2016, 05:23:05) 
[GCC 4.9.2] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import signalfx
>>> signalfx.SignalFx().ingest("asdf")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.5/site-packages/signalfx/__init__.py", line 98, in ingest
    from . import ingest
  File "/usr/local/lib/python3.5/site-packages/signalfx/ingest.py", line 16, in <module>
    from .generated_protocol_buffers \
  File "/usr/local/lib/python3.5/site-packages/signalfx/generated_protocol_buffers/signal_fx_protocol_buffers_pb2.py", line 111, in <module>
    has_default_value=False, default_value=unicode("", "utf-8"),
NameError: name 'unicode' is not defined
```

**Comment by mpetazzoni** [comment]
Ah, I guess the generated code from the protocol buffers is not for Python 3. Let me fix that.

**Comment by mpetazzoni** [comment]
See #32

**Comment by mpetazzoni** [comment]
Should be fixed in v1.0.7.
